subsys/usb/device/usb_descriptor.c

```diff
@@ -355,6 +355,32 @@ static void usb_fix_ascii_sn_string_descriptor(struct usb_sn_descriptor *sn)
 	memcpy(sn->bString, runtime_sn, runtime_sn_len);
 }
 
+static void usb_desc_update_mps0(struct usb_device_descriptor *const desc)
+{
+	struct usb_dc_ep_cfg_data ep_cfg = {
+		.ep_addr = 0,
+		.ep_mps = USB_MAX_CTRL_MPS,
+		.ep_type = USB_DC_EP_CONTROL,
+	};
+	int ret;
+
+
```

Comment:
I'm not sure if I hadn't seen it before or you add it right now, but there is one additional empty line. Since it's not relevant (only coding style) it can be approved. You can change it if you want, for me it's fine.